core/src/main/java/com/linecorp/armeria/server/file/NonExistentHttpFile.java

```diff
@@ -69,7 +69,8 @@ public HttpService asService() {
             }
 
             if (location == null) {
-                return HttpResponse.of(HttpStatus.NOT_FOUND);
+                return HttpResponse.of(HttpStatus.NOT_FOUND, MediaType.PLAIN_TEXT_UTF_8,
+                                       "Location is not set.");
             }
```

Comment:
I added it because I think it's convenient for us to have a hint if we don't have location information.

Reply:
From the client's point of view, I don't see they understand what `Location` means in `404 Not Found` response.

Reply:
Thank you for your feedback. ☺️
I think it would be nice to have a hint to specify the location, but I guess a 404 would make the intent clear.

If you want to be kind, how about a hint that no file information is passed?  If you want to provide a message, `doesn't include file information` would be nice.